Source/WTF/wtf/WeakPtr.h

```diff
@@ -34,8 +35,8 @@
 namespace WTF {
 
 struct EmptyCounter {
-    static void increment() { }
-    static void decrement() { }
+    void increment() { }
+    void decrement() { }
 };
```

Comment:
Since this counter thing is only useful in testing, we can put this in the specialized version of WeakPtrImpl
the same way we put EventTargetData in WeakPtrImpl.

Reply:
Done.